src/utils/HTMLUtils/index.js

```diff
@@ -13,13 +13,11 @@ const getInnerHTML = (elem) =>
 
 const removeDash = (elem) => replace(/-/g, "_", elem);
 
-const unescapePiping = (value, isMultipleChoiceValue) => {
+const unescapePiping = (value) => {
   let updatedValue;
-  if (!isMultipleChoiceValue) {
-    updatedValue = replace(/&apos;/g, `&#39;`, value);
-  } else {
-    updatedValue = replace(/&apos;/g, `\u2019`, value);
-  }
+
+  updatedValue = replace(/&apos;/g, `\u2019`, value);
+
   updatedValue = updatedValue.trim(); //remove leading and trailing spaces
 
   return updatedValue;
```

Comment:
As the `if` and `else` blocks have been removed, `updatedValue` no longer has to be redefined, and all of the `unescapePiping` function's code can be replaced with:
```
  return replace(/&apos;/g, `\u2019`, value).trim();
```